nfs-client/deploy/deployment.yaml

```diff
@@ -0,0 +1,31 @@
+kind: Deployment
+apiVersion: extensions/v1beta1
+metadata:
+  name: nfs-client-provisioner
+spec:
+  replicas: 1
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      labels:
+        app: nfs-client-provisioner
+    spec:
+      containers:
+        - name: nfs-client-provisioner
+          image: quay.io/jackieli/nfs-client-provisioner:v1
```

Comment:
we should push it to quay.io/external_storage too. I haven't set up an auto build trigger or anything like that but that's where efs, etc. are currently.